docs/src/events.md

```diff
@@ -102,8 +97,7 @@ await popup.GotoAsync("https://wikipedia.org");
 
 ## Adding/removing event listener
 
```

Comment:
Replace `Wait for popup window:` with "Wait for popup window using [`method: Page.waitForEvent`]:"